app/src/main/java/com/example/harvestfresh/MainActivity.java

```diff
@@ -1,15 +1,51 @@
 package com.example.harvestfresh;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
 
 import android.os.Bundle;
+import android.view.MenuItem;
+
+import com.example.harvestfresh.fragments.HomeFragment;
```

Comment:
Nit: At some point, you might want to change your package name to not have the word "example" in it, just for polish factor. com.harvestfresh or maybe even com.jefrin.harvestfresh is fine